package.json

```diff
@@ -14,6 +14,8 @@
     "dev:amo:no-proxy": "better-npm-run dev:amo:no-proxy",
     "dev:disco": "better-npm-run dev:disco",
     "eslint": "eslint .",
+    "flow:check": "flow check",
+    "flow:dev": "chokidar .flowconfig flow/ src/ tests/ -i flow/logs/flow.log -c 'flow status' --initial",
```

Comment:
Yep, it's safe. "In addition to the shell's pre-existing PATH, [npm run](https://docs.npmjs.com/cli/run-script) adds node_modules/.bin to the PATH provided to scripts."